docs/CONTRIBUTING.adoc

```diff
@@ -0,0 +1,31 @@
+= Contributing
```

Comment:
I'd expect to see the rules that we set in the #121

Reply:
Will add a style guide.

`we need to make sure that changes done by any contributor are understandable for everybody.`
About this one I'm thinking and also think it is at least partially addressed by what is provided.

I can see that people can help by providing the title so that maintainers dont have to change it for themselves.
Other things i can see in the guidelines like:
`we will need to make sure that the releases are correctly tagged`
or
`have proper changelogs for node runners`
is for maintainers of the project as far as i understand. Should this be provided here also?

Reply:
No. Probably just info about using conventional commits for pr names, using semver and writing tests

Reply:
Understood. Incoming.

Conventional commits for pr names is mentioned in line 18 of [CONTRIBUTING.adoc.] (https://github.com/RoboRambo/HydraDX-node/blob/add_contributing_guidelines/docs/CONTRIBUTING.adoc).

Reply:
We discussed testing in regard to versioning, not sure what to add here. Maybe that test should be provided?